recipes/filprofiler/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "filprofiler" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pythonspeed/filprofiler/archive/{{ version }}.tar.gz
+  sha256: bd42134063a106e61e69b7b28fc85a9777bb476c05338038fed22065ee66ffb6
+
+build:
+  number: 0
+  skip: True  # [win or py<36]
+  script: |
+      set -euo pipefail;
+      make target/release/libpymemprofile_api.a;
+      export PIP_LOG=/dev/stdout;
+      if [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
+        # Disable aligned_alloc(). Conda uses ancient glibc headers from 2010,
+        # where aligned_alloc is defined as inline function, rather than as
+        # overridable symbol.
+        sed -i 's/SYMBOL_PREFIX.aligned_alloc/SYMBOL_PREFIX(no_aligned_alloc_on_conda/' filprofiler/_filpreload.c
+      fi;
+      {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
+    - rust
+  host:
+    - python
+    - pip
```

Comment:
Shouldn't rust be in host?

Reply:
I went and read Conda docs, and... I don't know? It's a compiler, like the C compiler.

I found this existing example, where `rust` is in `build`:

https://github.com/conda-forge/ripgrep-feedstock/blob/master/recipe/meta.yaml#L16

Reply:
hmm that seems like it should be in host too. That is less of a deal for ripgrep.  Here, you want the rust binaries to be linked to python though?

Reply:
I'm building a shared library that is partially Rust and partially C and gets built by distutils (and calls Python/C APIs) due to Reasons. The C compiler is in `build`, so I figured the Rust compiler should be in `build` too? But the documentation did not really make sense to me in terms of what the difference is, if you think it should be `host` I can change it.

Reply:
OK, done